modules/ROOT/pages/database-administration/composite-databases/concepts.adoc

```diff
@@ -4,60 +4,35 @@
 [[composite-databases-concepts]]
 = Concepts
 
-A Composite database is a special type of database.
+A composite database is a special type of database.
 Composite databases are the means to access this partitioned data or graphs with a single Cypher query.
 
 Composite databases *do not store data* independently.
-They contain _database aliases_ that target the local or remote databases (the so-called _constituents_) that constitute the fabric setup.
+They contain _database aliases_ that target the local or remote databases (the so-called _constituents_).
 Local database aliases target databases within the same DBMS, while remote database aliases target databases from another Neo4j DBMS.
 For more information, see xref:database-administration/aliases/manage-aliases-composite-databases.adoc[].
 
 Composite databases are managed using Cypher administrative commands.
-Therefore, you can create them as any other database in Neo4j standalone and cluster deployments without needing to deploy a dedicated proxy server (as with Fabric in Neo4j 4).
 For a detailed example of how to create a Composite database and add database aliases to it, see xref:database-administration/composite-databases/querying-composite-databases.adoc[Set up and query composite databases].
 
-Composite databases cannot guarantee compatibility between constituents from different major versions of Neo4j.
-Therefore, all constituents should belong to the same major version.
+Composite databases cannot guarantee compatibility between constituents from different Neo4j versions.
+Constituents from versions without breaking changes should work fine, apart from newly-added features.
 If a new feature is introduced, its availability will be limited to the intersection of features available for all constituents.
```

Comment:
Technically, this would be more correct, if we want to be precise:
```suggestion
If a new feature is introduced, its availability will be limited to the intersection of features available on the DBMS where the composite database is defined, as well as all of its constituents.
```